infra/hasura/.env.example

```
@@ -1,9 +1,16 @@
-# Hasura
-HASURA_GRAPHQL_ENDPOINT=http://localhost:8080
-HASURA_GRAPHQL_ADMIN_SECRET=myadminsecretkey
+# ── PostgreSQL ───────────────────────────────────────────
+POSTGRES_USER=#postgresuser
+POSTGRES_PASSWORD=#postgrespassword
+POSTGRES_DB=#postgresdb
+POSTGRES_HOST=#localhost
+POSTGRES_PORT=#5432
+PG_DATABASE_URL=## For Hasura CLI and other tools that need a connection string
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PG_DATABASE_URL key should go before the POSTGRES_DB key

(UnorderedKey)